environment-model/environment-model_ontology.ttl

```diff
@@ -1,17 +1,16 @@
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @prefix environment-model: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environment-model/> .
-@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 environment-model: a owl:Ontology ;
-    rdfs:label "ontology definition for environment-model"@en ;
+    rdfs:label "ontology definition for EnvironmentModel"@en ;
     dcterms:contributor "Mirco Nierenz (TG)" ;
     owl:versionInfo "0.1"^^xsd:float .
```

Comment:
^^xsd:float can be removed

Reply:
outdated? see branch environment-model and 
https://github.com/GAIA-X4PLC-AAD/ontology-management-base/pull/44

Reply:
still the same in the code.